optimum/exporters/onnx/model_configs.py

```diff
@@ -1085,3 +1086,37 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
             common_inputs["encoder_outputs"] = {0: "batch_size", 1: "encoder_sequence_length"}
 
         return common_inputs
+
+
+# This is extremly ugly
+class NormalizedSamConfig(NormalizedVisionConfig):
+    VISION_CONFIG = "vision_config"
+
+    def __getattr__(self, attr_name):
+        if self.VISION_CONFIG is not None and attr_name.upper() in dir(NormalizedVisionConfig):
+            attr_name = f"{self.VISION_CONFIG}.{attr_name}"
+        return super().__getattr__(attr_name)
+
```

Comment:
Cant use use vision encoder decoder config?

Reply:
It works, thanks for the hint!